sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -1,25 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure.Storage.Blobs client library</AssemblyTitle>
-    <AssemblyName>Azure.Storage.Blobs</AssemblyName>
-    <RootNamespace>Azure.Storage.Blobs</RootNamespace>
     <DefineConstants>BlobSDK;$(DefineConstants)</DefineConstants>
     <PackageTags>Microsoft Azure Storage Blobs, Microsoft, Azure, Blob, StorageScalable, azureofficial</PackageTags>
     <Description>
       This client library enables working with the Microsoft Azure Storage Blob service for storing binary and text data.
-
-      For this release see notes:
-      - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs/README.md and
-      - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs/changelog.txt
-
-      in addition to the breaking changes:
-      - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt
-
-      Microsoft Azure Storage team's blog - http://blogs.msdn.com/b/windowsazurestorage/
+      For this release see notes - https://github.com/Azure/azure-storage-net/blob/master/Blob/README.md and https://github.com/Azure/azure-storage-net/blob/master/Blob/Changelog.txt
+      in addition to the breaking changes https://github.com/Azure/azure-storage-net/blob/master/Blob/BreakingChanges.txt
+      Microsoft Azure Storage quickstarts and tutorials - https://docs.microsoft.com/en-us/azure/storage/
+      Microsoft Azure Storage REST API Reference - https://docs.microsoft.com/en-us/rest/api/storageservices/
+      REST API Reference for Blob Service - https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api
     </Description>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Common\src\Shared\**\*.cs" />
```

Comment:
Add a TO-DO to move this back when the build system can support it?

This is going to make VS a little weird -- it doesn't like to pick up new files like this.

Reply:
We should dig into this and figure out whether we actually need the Shared files.  Without the pipeline, there's so much less code being shared that we might be able to find it a single home somewhere.

Reply:
Worth discussing.  There are some shared concepts like SAS generation that have slight service-specific differences that need to be catered to, which probably should *not* lead to types that can be shared across services, but which are complex enough that you want to keep from having to maintain duplicate code.  

Unlike the Common library, which is for things which can/should be shareable.